src/theme/animationTimings.js

```diff
@@ -0,0 +1,9 @@
+const listTimings = {
```

Comment:
I think we can make these timing values a bit more generic. something like:
- enter: 100
- exit: 300
- enterSpecial: 500